enterprise/backend/src/metabase_enterprise/serialization/names.clj

```diff
@@ -24,10 +24,16 @@
 
 (def ^:private root-collection-path "/collections/root")
 
+(def ^:private label-limit 100)
```

Comment:
I wonder if we could/should let this be overridden by a property or env variable. That would in some sense remove the arbitrariness.

Reply:
Sure but that's adding quite a bit of functionality for a bugfix on a deprecated feature.

Reply:
I agree with John; it's not worth the effort for a very rare bug on v1.